ucs/database.cpp

```diff
@@ -55,6 +55,8 @@ extern std::string GetMailPrefix();
 extern ChatChannelList *ChannelList;
 extern uint32          MailMessagesSent;
 
+std::vector<std::string> BlockedChannelNames;
```

Comment:
We should avoid having this be a global

This should live as a member scoped variable somewhere where it makes sense to have the responsibility be housed.

My suggestion would be to have it live on the `ChatChannelList` class since this pertains to **many** channels.

Private `m_blocked_channel_names`

Then create getter / setter to mutate the vector publicly

Reply:
Updated